website/views/pages/get-started.ejs

```diff
@@ -26,27 +33,31 @@
         <p class="comment"># Run a local demo of the Fleet server</p>
         <p class="command">sudo fleetctl preview</p>
       </div>
-        <p class="mb-0"><img alt="A small circle with an I inside of it" style="width: 16px; display: inline; margin-right: 8px; margin-bottom: 3px" src="/images/info-16x16@2x.png">Windows users can omit <code>sudo</code> .</p>
+      <p class="mb-0"><img alt="A small circle with an I inside of it"
+          style="width: 16px; display: inline; margin-right: 8px; margin-bottom: 3px"
+          src="/images/info-16x16@2x.png">Windows users can omit <code>sudo</code> .</p>
     </div>
     <div style="padding-top: 60px;">
       <h2 class="mb-3">3. Log in to Fleet</h2>
-        <p>The Fleet UI is now available at <a href="http://localhost:1337" target="_blank">http://localhost:1337</a>.</p>
-        <p class="mb-2"><strong>Email:</strong> admin@example.com</p>
-        <p><strong>Password:</strong> admin123#</p>
+      <p>The Fleet UI is now available at <a href="http://localhost:1337" target="_blank">http://localhost:1337</a>.</p>
```

Comment:
nice, thank you for the attention to detail on this indentation fix

Reply:
np!